### DISCUSSION TITLE
Configuration Option for attach_stacktrace in Sentry-Native Not Implemented

### DISCUSSION THREAD

**Discussion by EvgenPervenenko** [getsentry/sentry-native#1095] [Ideas]
Hi, community, are there any plans to implement the attach_stacktrace configuration option API in the Sentry SDK?

**Comment by supervacuus** [comment]
There are no plans, but the topic wasn't discussed either (I wasn't aware this option exists in other SDKs). Let's convert this to a feature request to gather interest.
